ethcore/src/state/mod.rs

```diff
@@ -853,27 +849,29 @@ impl<B: Backend> State<B> {
 
 	// load required account data from the databases.
 	fn update_account_cache(require: RequireCache, account: &mut Account, state_db: &B, db: &HashDB) {
-		match (account.is_cached(), require) {
```

Comment:
is changing this actually a cleanup or improvement? I think it was fine before

Reply:
IMHO it's cleaner now (one less identation, clear indication that already cached accounts and `RequireCache::None` is not handled at all).

Reply:
@tomusdrw I was about to write exactly the same thing ;)